### PR TITLE
bw: add page

### PR DESCRIPTION
**Pull request by NithyaNatarajan**
This PR adds tldr for `https://github.com/bitwarden/cli/`

Signed-off-by: Nithya Natarajan <nithyanatarajn@gmail.com>

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/bw.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# bw
+
+> A CLI to access and manage a Bitwarden vault.
+
+- To login to a Bitwarden user account:
```
Comment (pxgamer) [inline pages/common/bw.md]: Perhaps remove the `To` from each example:
```suggestion
- Log in to a Bitwarden user account:
```

File: pages/common/bw.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`bw login`
+
+- To logout of a Bitwarden user account:
```
Comment (pxgamer) [inline pages/common/bw.md]:
```suggestion
- Log out of a Bitwarden user account:
```

File: pages/common/bw.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`bw logout`
+
+- To search and display items from Bitwarden vault:
```
Comment (pxgamer) [inline pages/common/bw.md]:
```suggestion
- Search and display items from Bitwarden vault:
```

File: pages/common/bw.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`bw list items --search {{github}}`
+
+- To display a particular item from Bitwarden vault:
```
Comment (pxgamer) [inline pages/common/bw.md]:
```suggestion
- Display a particular item from Bitwarden vault:
```

File: pages/common/bw.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`bw get item {{github}}`
+
+- To create a folder in Bitwarden vault:
```
Comment (pxgamer) [inline pages/common/bw.md]:
```suggestion
- Create a folder in Bitwarden vault:
```

Also, just out of interest, does it always require a base64-encoded JSON string as input? 🙂

Comment (agnivade) [reply pages/common/bw.md]: Also, is it necessary for the text to be in json ?

Comment (pxgamer) [reply pages/common/bw.md]: Hmm, looks like it does. :thinking_face:

https://help.bitwarden.com/article/cli/#create

**Comment by NithyaNatarajan** [comment]
@agnivade, will accommodate the suggestions.


@pxgamer and @agnivade, yup, as mentioned already by @pxgamer, the `bw create` always takes an encoded JSON. It could be because the same command is used to add a folder and an item (username, password)

**Comment by NithyaNatarajan** [comment]
@agnivade Sure, will not amend commits next time 👍 Thanks for merging

**Pull request merged**